web/public/mgmt/config.yml

```diff
@@ -3074,6 +3074,15 @@ collections:
             fields:
               - { label: Beta Main Text, name: betaMainText, widget: string }
               - { label: Beta Modal Button Text, name: betaModalButtonText, widget: string }
+          - label: SEO and Accessibility Page Titles
+            name: seoAndAccessibilityPageTitles
+            collapsed: true
+            widget: object
+            summary: "{{seoAndAccessibilityPageTitles}}"
+            fields:
+              - { label: Home Page Title, name: homePageTitle, widget: string }
+              - { label: Dashboard Page Title, name: dashboardPageTitle, widget: string }
+              - { label: Business Profile Page Title, name: profilePageTitle, widget: string }
```

Comment:
Yes we probably need all of these. The word dashboard and home don't appear on their pages. 

And the Business Profile does have the words Business Profile but it has markdown header hashtags in front of it and I don't think it wou
ld be good to remove those and I'm not sure it woudl be worth the refactor 
<img width="1173" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/540be234-fbcd-4ab5-89b0-e78ef0569693">

Reply:
I tend to agree with you here. I'd like to have config-style documents as purely markup-free strings and other primitives. I'd create a separate chore and make that change, assuming your story requires Design's review anyway.

Reply:
I'll write a follow up ticket after lunch for getting rid of the whole H3 in a json file thing and finding similar issues as a refactor ticket

Reply:
https://www.pivotaltracker.com/story/show/186435464